package.json

```diff
@@ -2,14 +2,14 @@
   "name": "user-activity-watcher",
   "version": "1.0.0",
   "description": "Watches user activity on issues, sends reminders on deadlines, and unassign inactive users.",
-  "main": "src/worker.ts",
+  "main": "src/index.ts",
   "author": "Ubiquity DAO",
   "license": "MIT",
+  "type": "module",
```

Comment:
What's this for?

Reply:
This is very likely to be compatible with ESM libraries, like `octokit/graphql`.